src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -28,7 +28,6 @@
           <div class="kuiLocalSearch">
             <input
               parse-query
-              input-focus
```

Comment:
Ultimately, if/when https://github.com/elastic/kibana/pull/10976 gets in, we could move to something more like how Google Drive does this:

![image](https://cloud.githubusercontent.com/assets/1178348/24634357/62a2e32a-1882-11e7-92c3-17dd192e88c2.png)

Notice how there's a "Learn More" button in the bottom of the pop-up. Thoughts?